package.json

```diff
@@ -15,8 +15,8 @@
     "@capacitor/android": "^2.4.7",
     "@capacitor/core": "^2.4.7",
     "@ionic/core": "6.0.3",
```

Comment:
I think we may need to upgrade core as well. Please check

Reply:
Yes, sir. Checked, upgrade is required. Upgrading it to 6.2.6 from 6.0.3.

Reply:
Downgraded Ionic core to 6.1.15 as discussed.